src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -1,5 +1,14 @@
 ({
     /*******************************************************************************************************
+     * @description called at onInit to detect if Locker Service is enabled, warning if so.
+     */
+    warnOnLockerService: function(component) {
+        if (component.$lskey != null) {        
+            this.displayUIMessage(component, "LockerService is activated which prevents this page from working correctly.  Under Setup, search for Critical Updates, and deactivate the critical update 'Enable Lightning LockerService Security'. ", "divUIMessageContainer");
```

Comment:
correct.  short lived, so no label was intentional. 

@jvarese are you ok with the message "LockerService is activated which prevents this page from working correctly.  Under Setup, search for Critical Updates, and deactivate the critical update 'Enable Lightning LockerService Security'."?  or do you have any edits you'd like to it?